sgv2-filter.meta.js

```diff
@@ -3,10 +3,11 @@
 // @namespace   https://github.com/GarionCZ/sgv2-filter
 // @description Giveaway filter for SteamGifts v2
 // @author      Garion
-// @include     http://www.steamgifts.com/*
+// @include     http://www.steamgifts.com/giveaways*
```

Comment:
I thought you'd want to modify it then :) The best approach in my opinion is to include the whole SG and then just exclude the pages where reported broken.